libs/openFrameworks/video/ofVideoBaseTypes.h

```diff
@@ -209,6 +209,10 @@ class ofBaseVideoPlayer: virtual public ofBaseVideo{
 	/// \sa loadAsync()
 
 	virtual bool load(const of::filesystem::path & fileName) = 0;
```

Comment:
I think this might need to be not pure virtual as inherited classes from older addons won't have this implemented